docs/content/latest/deploy/docker/docker-swarm.md

```diff
@@ -227,7 +230,8 @@ $ docker service create \
 --publish 9000:9000 \
 yugabytedb/yugabyte:latest /home/yugabyte/bin/yb-tserver \
 --fs_data_dirs=/mnt/data0 \
---tserver_master_addrs=yb-master1:7100,yb-master2:7100,yb-master3:7100
+--rpc_bind_addresses=0.0.0.0:9100 \
```

Comment:
```suggestion
--rpc_bind_addresses=task.yb-master:9100 \
```

Reply:
@baba230896 That setting is applicable to both yb-master and yb-tserver. It's not explicitly stated in the docs, so I will fix that.

Reply:
@stevebang If we pass this "tasks.yb-master1:9100" to the "yb-tserver" service's rpc_bind_address then deployment will fail because "tasks.yb-master1" points towards the Container IP of yb-master1 service and yb-tserver service instance doesn't have this IP in its network configuration.